content/en/docs/develop/porting.md

```diff
@@ -41,64 +41,63 @@ For this tutorial, we will be targeting the network utility program [`iperf3`](h
 `iperf3` is a benchmarking tool, and is used to determine the bandwidth between a client and server.
 It is an excellent application to be run as a Unikernel for several reasons:
 
- * It can run as a "server-type" application, receiving and processing requests for clients;
- * It is a standalone tool which does one thing;
- * It's [GNU Make](https://www.gnu.org/software/make/) and C-based;
- * And it's quite useful :)
+1. It can run as a "server-type" application, receiving and processing requests for clients;
+1. It is a standalone tool which does one thing;
+1. It's [GNU Make](https://www.gnu.org/software/make/) and C-based;
+1. And it's quite useful :)
 
 Understanding some aspects of how an application works, particularly how it is built, is required when bringing it to Unikraft.
 Usually during the porting process we also end up diving through the source code, and in the worst-case scenario, have to make a change to it.
 More on this is covered in the [Patching](./docs/develop/patching) section.
```

Comment:
I think we don't have any longer this section.

Reply:
That will be covered by #116, it's not the purpose of this pr.